snap/snapcraft.yaml

```
@@ -132,12 +128,11 @@ parts:
       snap install node --channel=18/stable --classic
 
   wazuh-dashboard:
-    plugin: nil
+    source: https://people.canonical.com/~arturoseijas/wazuh-dashboard_4.9.2-1_amd64.deb
```

```suggestion
    source: https://packages.wazuh.com/4.x/apt/pool/main/w/wazuh-indexer/wazuh-indexer_4.9.2-1_amd64.deb
    source-checksum: sha512/dc3135ca68376fb53a5b2377ec17d22e701b2bf944fc45596a33a378d810f5a9892aa2e53eda631b6ee4fa9c9ab838710b4953e54616ca61e5b339bdbdc777fe
```
I think it might be better to download it from Wazuh, but pin the checksum of the download to ensure it doesn't change.

---

It's not the same package. This one is customi<ed for us so that we can snap it